docs/en/stack/ml/get-started/ml-gs-results.asciidoc

```diff
@@ -49,8 +49,10 @@ display only a single time series.
 
 The blue line in the chart represents the actual data values. The shaded blue
 area represents the bounds for the expected values. The area between the upper
-and lower bounds are the most likely values for the model. If a value is outside
-of this area then it can be said to be anomalous.
+and lower bounds are the most likely values for the model, using a 95%
+confidence level. That is to say, there is a 95% chance of the actual value
+falling within these bounds. If a value is outside of this area then it can be
+said to be anomalous.
```

Comment:
We don't always raise an anomaly in this case although usually do. Perhaps the wording here can be tweaked to avoid questions like "we didn't get an anomaly but you said we should"
```suggestion
falling within these bounds. If a value is outside of this area then it will usually
be identified as anomalous.
```